docs/core/distribution-packaging.md

```diff
@@ -198,6 +198,57 @@ Most distributions require all artifacts to be built from source. This has some
 
 Multiple `dotnet-sdk` packages may provide the same files for the `NuGetFallbackFolder`. To avoid issues with the package manager, these files should be identical (checksum, modification date, and so on).
 
+### Debug packages
+
+Debug content should be packaged in debug-named packages that follow the .NET package split described previously in this article. For instance, debug content for the `dotnet-sdk-[major].[minor]` package should be included in a package named `dotnet-sdk-dbg-[major].[minor]`. You should install debug content to the same location as the binaries.
```

Comment:
Any thoughts on making this more flexible to comply with additional distro conventions? Fedora, for example, seems to prefer `debug` over `dbg`:

```
$ podman run -it fedora:39
# dnf list -q '*debug*' | wc -l
219
# dnf list -q '*dbg*' | wc -l
4
```

Reply:
Part of the value of these guidelines is for different distros to use the same names.

These are all "recommendations". A maintainer can still chose to do something different.